Automattic/jetpack
# Plans: Update new Backup/Security product slugs

Fixes `1200412004370260-as-1200738499437035`.

#### Changes proposed in this Pull Request:

* Update all tier-1 `JETPACK_SECURITY` and `JETPACK_BACKUP` slugs to be appended with `_T1`.
* Update all tier-2 `JETPACK_SECURITY_PRO` and `JETPACK_BACKUP_PRO` slugs to replace the `_PRO` suffix with `_T2`.
* Update above yearly-variation product slugs to have a `_YEARLY` suffix.

#### Jetpack product discussion

`p1HpG7-cI6-p2`
`p1HpG7-cyv-p2#comment-48134`

#### Does this pull request change what data or activity we track or use?

Nope 👍 

#### Testing instructions:

Instructions are identical to #20344, copied here (with updated PR reference) for convenience:

* Verify all the new slugs match perfectly with the ones added in Automattic/wp-calypso#55475.
* All tests should still pass (e.g., pnpm test-adminpage).
* All pages should still function as they currently do, with no changes. This PR should have no implications for the user experience.
* Specifically check that all product/plan icons on the At A Glance page display correctly.
